core/src/test/scala/org/apache/spark/shuffle/ShuffleBlockPusherSuite.scala

```diff
@@ -56,8 +56,6 @@ class ShuffleBlockPusherSuite extends SparkFunSuite with BeforeAndAfterEach {
     when(dependency.partitioner).thenReturn(new HashPartitioner(8))
     when(dependency.serializer).thenReturn(new JavaSerializer(conf))
     when(dependency.getMergerLocs).thenReturn(Seq(BlockManagerId("test-client", "test-client", 1)))
-    conf.set("spark.shuffle.push.based.enabled", "true")
-    conf.set("spark.shuffle.service.enabled", "true")
```

Comment:
Dont we need to enable push based shuffle ? via `spark.shuffle.push.enabled` ?

Reply:
No we don't need to. The tests here are for `ShuffleBlockPusher` which gets created only when shuffle push is enabled. These test are just directly creating `ShuffleBlockPusher`.

Earlier all the code in `ShuffleBlockPusher` was in `ShuffleWriter` which would initiate push when it was enabled so we needed these configs in the tests for `ShuffleWriter`. However, during the review of #30312 we moved all this functionality to its own class and the tests were moved as well. Just missed removing these configs from this test suite.